tests/forte/data/data_store_serialization_test.py

```diff
@@ -0,0 +1,585 @@
+                                None,
+                                None,
+                            ],
+                            [
+                                55,
+                                70,
+                                1234567,
+                                "ft.onto.base_ontology.Sentence",
+                                None,
+                                None,
+                                "Negative",
+                                None,
+                                "Class D",
+                            ],
+                            [
+                                60,
+                                90,
+                                100,
+                                "ft.onto.base_ontology.Sentence",
+                                "student",
+                                2,
+                                "Positive",
+                                None,
+                                "class1",
+                            ],
+                            [
+                                65,
+                                90,
+                                5000,
+                                "ft.onto.base_ontology.Sentence",
+                                "TA",
+                                3,
+                                "Positive",
+                                None,
+                                "class2",
+                            ],
+                        ],
+                    ),
+                },
+            )
+            self.assertEqual(
+                temp._DataStore__entry_dict,
+                {
+                    1234: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][0],
+                    3456: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][1],
+                    4567: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][2],
+                    5678: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][3],
+                    7890: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][4],
+                    9999: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][0],
+                    1234567: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][1],
+                    100: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][2],
+                    5000: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][3],
+                },
+            )
+            print(f"serialization with save attribute time cost {b-a}")
```

Comment:
remove the `print` statements, and no need to add the time profiling in test cases.

Reply:
fixed